### DISCUSSION TITLE
Rough conductor material

### DISCUSSION THREAD

**Discussion by johnson-c** [mitsuba-renderer/mitsuba3#1043] [Q&A] [answered]
How would I handle having multiple pieces of CAD with different sizes and the same roughness. I've skimmed through the Walter 2007 paper but I don't see anything about relating the alpha parameter back a physical unit.

There is this from the docs "When the Beckmann distribution is used, this parameter is equal to the root mean square (RMS) slope of the microfacets. " but what are the units of RMS? I'm guessing this is related to the bounding box of the object?

**Comment by njroussel** [comment]
Hi @johnson-c 

I might be misunderstanding your questions.

The micro-scale geometry (microfacets) is modeled as a distribution of normals. The alpha values are just parameters of those distributions, they therefore don't have any physical unit tied to them, I believe.
The microfact-model assumes that we can sample a surface normal from that distribution to approximate the local roughness. We are essentially picking random directions without ever assuming any size of the individual facets that we supposedly hit (if anything, we consider them as infinitely small).
Does this make sense?

**Reply by johnson-c** [reply]
Hi @njroussel 

Thanks for taking the time to answer. This makes sense in how it would get implemented in the code. I might be misunderstanding entirely, I guess my question is how to relate alpha (a dimensionless number) to something physical like surface roughness RMS which would be units of micrometers.

**Reply by njroussel** [reply]
I personally don't know if there is any existing literature on this.

Intuitively, there should be some kind of a scale factor that would need to be taken into account: the `ggx`/`beckmann` roughness would look the same whether your viewpoint is close or far from the material which is not what you want.

**Reply by johnson-c** [reply, marked as answer]
Hi @njroussel 

I found a paper that experimentally measured the alpha parameter and related it to a measured roughness (its basically exactly what I'm trying to investigate, reflections off of tungsten surfaces in a fusion device). Though its still not entirely clear to me if this could have been calculated without needing to tie it to an experimental measurement.

https://www.sciencedirect.com/science/article/pii/S2352179122001247
